spec/models/keyword_spec.rb

```diff
@@ -5,11 +5,32 @@
 RSpec.describe Keyword do
   subject(:keyword) { build(:keyword) }
 
+  let(:work_version) { build(:work_version) }
```

Comment:
If `work_version` is only used in the whitespace tests, you could consider moving the `let` statement down into the relevant `describe` block.